src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -174,7 +174,9 @@ def _grant_access(cmd, resource_group_name, name, duration_in_seconds, is_disk,
 
 
 def _is_linux_os(vm):
-    os_type = vm.storage_profile.os_disk.os_type if vm.storage_profile.os_disk.os_type else None
+    os_type = None
```

Comment:
if os_type is None, will it be updated to None in later call? what's service behavior, ignore or update?

Reply:
This function does not update any property. It returns a bool value.
If we call PUT again to an existing resource, None means ignore. No effect.